Components/Toast/toast.html

```diff
@@ -0,0 +1,94 @@
+                    <li class="sidebar-link-item "><a href="./../Card/card.html ">Card</a></li>
+                    <li class="sidebar-link-item "><a href="./../Image/image.html ">Image</a></li>
+                    <li class="sidebar-link-item "><a href="./../Input/input.html">Input</a></li>
+                    <li class="sidebar-link-item "><a href="./../Lists/lists.html ">Lists</a></li>
+                    <li class="sidebar-link-item "><a href="./../Toast/toast.html ">Toast</a></li>
+                </ul>
+            </div>
+        </aside>
+
+        <main>
+            <!-- Toast component start from docs -->
+            <div class="sakhi-component-container ">
+                <h2 class="component-heading " id="avatar ">Toast</h2>
+                <p class="component-desc "> Toast is mostly used to show quick messages on process after some actions took place in application. Toast can be used to show error, success, warning messages.</p>
+                <div class="component-container ">
+                    <div class="simple-toast-body">
+                        Hey there! it's a toast message.
+                        <button type="button" class="toast-close-btn"></button>
+                    </div>
+                    <div class="simple-toast-body toast-success">
+                        Successfully! downloaded.
+                        <button type="button" class="toast-close-btn"></button>
+                    </div>
+                    <div class="simple-toast-body toast-error">
+                        something went wrong!
+                        <button type="button" class="toast-close-btn"></button>
+                    </div>
+                    <div class="simple-toast-body toast-warning">
+                        Please check before submit..
+                        <button type="button" class="toast-close-btn"></button>
+                    </div>
+                </div>
+                <div>
+                    <iframe src="https://www.thiscodeworks.com/embed/62037017234b8a0015700c72" style="width: 100%; height: 194px;" frameborder="0"></iframe>
+                </div>
+                <h2 class="component-heading " id="avatar ">Example of Toast</h2>
+                <p class="component-desc "> Push notifications to your visitors with a toast, a lightweight and easily customizable alert message.</p>
+                <div class="component-container ">
+                    <div class="toast-body">
+                        Can't send photos. Retry in 5 seconds.
+                        <button class="retry-btn">RETRY</button>
+                        <span class="icon-toast">
+                            <i class="fas fa-times"></i>
+                        </span>
+                    </div>
+                </div>
+                <div>
+                    <iframe src="https://www.thiscodeworks.com/embed/62037b93234b8a0015700c85" style="width: 100%; height: 107px;" frameborder="0"></iframe>
```

Comment:
for height use rem units

Reply:
for iframe width and height is given default by thiscodeworks , i'm not changing it

Reply:
okay